src/build/mdn-comments.ts

```diff
@@ -47,66 +45,109 @@ function extractSummary(markdown: string): string {
     return sentenceMatch[0]; // Return the first full sentence
   }
 
-  return normalizedText.split(" ")[0] || ""; // Fallback: first word if no sentence found
+  const firstWord = normalizedText.split(" ")[0];
+  return firstWord || "";
```

Comment:
We should probably drop this, `RTCTrackEvent.track` only gets "The" currently. But it's fine to fix that in a followup.